md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/AppDeplymentQueueAPI.java

```diff
@@ -0,0 +1,226 @@
+package com.wipro.ats.bdre.md.rest;
+
+import com.wipro.ats.bdre.exception.MetadataException;
+import com.wipro.ats.bdre.md.api.util.AddJson;
+import com.wipro.ats.bdre.md.app.AppStore;
+import com.wipro.ats.bdre.md.beans.table.AppDeploymentQueue;
+import com.wipro.ats.bdre.md.beans.table.Process;
+import com.wipro.ats.bdre.md.beans.table.Properties;
+import com.wipro.ats.bdre.md.dao.AppDeploymentQueueDAO;
+import com.wipro.ats.bdre.md.dao.ProcessDAO;
+import com.wipro.ats.bdre.md.dao.PropertiesDAO;
+import com.wipro.ats.bdre.md.dao.jpa.AppDeploymentQueueStatus;
+import com.wipro.ats.bdre.md.dao.jpa.Users;
+import com.wipro.ats.bdre.md.rest.beans.ProcessExport;
+import com.wipro.ats.bdre.md.rest.util.BindingResultError;
+import com.wipro.ats.bdre.md.rest.util.DateConverter;
+import org.apache.log4j.Logger;
+import org.codehaus.jackson.map.DeserializationConfig;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by cloudera on 3/8/16.
+ */
+@Controller
+@RequestMapping("/adq")
+
+public class AppDeplymentQueueAPI {
+
+    private static final Logger LOGGER = Logger.getLogger(AppDeplymentQueueAPI.class);
+    @Autowired
+    AppDeploymentQueueDAO appDeploymentQueueDAO;
+    @Autowired
+    private ProcessDAO processDAO;
+    @Autowired
+    private PropertiesDAO propertiesDAO;
+
+    @RequestMapping(value = {"/", ""}, method = RequestMethod.POST)
+    @ResponseBody
+    public
+    RestWrapper insert(@ModelAttribute("acq")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "insert" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)